index.html

```diff
@@ -13,31 +13,27 @@
     <link href="https://fonts.googleapis.com/css?family=Domine&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href ="normalize.css">
-    <link rel="stylesheet" href="main.css">
-=======
     <link rel="stylesheet" href="styles/main.css">
->>>>>>> refs/heads/master
+
     <title>Kevin's CSD120 Website</title>
 </head>
 <body>
     <header>
         <nav>
             <ul>
-                <li class="header"><a href="index.html">Home</a></li>
-                <li class="header"><a href="links.html">Links</a></li>
-                <li class="header"><a href="glossary.html">Gallery</a></li>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="links.html">Links</a></li>
+                <li><a href="glossary.html">Gallery</a></li>
```

Comment:
No need for these classes to achieve what's requested in the lab.  use a selector like `header nav li` instead of `.header`